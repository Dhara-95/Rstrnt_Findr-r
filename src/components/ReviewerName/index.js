import React from "react";

const ReviewerName = (props) => {
  const { name } = props;

  return <p className="text-grey text-xl">{`-- ${name}`}</p>;
};

export { ReviewerName };
