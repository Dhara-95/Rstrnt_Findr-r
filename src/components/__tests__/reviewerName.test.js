import React from "react";
import renderer from "react-test-renderer";

import { ReviewerName } from "../ReviewerName";

it("displays the reviewer's name correctly", () => {
  const tree = renderer.create(<ReviewerName name={"name"} />).toJSON();
  expect(tree).toMatchSnapshot();
});
