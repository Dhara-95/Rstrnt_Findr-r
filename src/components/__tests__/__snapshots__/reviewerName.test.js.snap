// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`displays the reviewer's name correctly 1`] = `
<p
  className="text-grey text-xl"
>
  â€” name
</p>
`;
